### ISSUE TITLE
Question about pretrained models

### ISSUE THREAD

**Issue by sarvasvarora** [chenyaofo/pytorch-cifar-models#7]
Hi, I was wondering how many epochs were the pre-trained models trained for? And what other hyperparameters (learning rate, optimizer, scheduler, etc.) were used? I'm particularly interested in ResNet-20. 
Thanks!
